### PR TITLE
[portaudio] Switch to vcpkg_from_github, minor format improvements

### PR DESCRIPTION
**Pull request by h3ndrk**
**Describe the pull request**

- #### What does your PR fix?  
  - Update line numbers of existing patch
  - Switch from `vcpkg_from_git()` to `vcpkg_from_github()`
  - Convert `portfile.cmake` from DOS line-endings to Unix line-endings

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  No changes, left untouched

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes (apart from the portfile line-endings)

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

**Review by PhoebeHui — COMMENTED** [review]
LGTM, @h3ndrk, thanks for your PR!

**Review by PhoebeHui — APPROVED** [review]

**Comment by Hoikas** [comment]
IMO `add-namespace.patch` is inappropriate. Upstream should be in charge of their exported targets. By adding this patch, we add a compatibility break between the vcpkg built portaudio and the upstream.

File: ports/portaudio/portfile.cmake [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
+vcpkg_fail_port_install(ON_TARGET "uwp")
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO PortAudio/portaudio
+    REF 147dd722548358763a8b649b3e4b41dfffbcfbb6
+    TAG v19.7.0
+    SHA512 0f56e5f5b004f51915f29771b8fc1fe886f1fef5d65ab5ea1db43f43c49917476b9eec14b36aa54d3e9fb4d8bdf61e68c79624d00b7e548d4c493395a758233a
+    PATCHES
+        fix-library-can-not-be-found.patch
+        add-namespace.patch
```
Comment (ras0219-msft) [inline ports/portaudio/portfile.cmake:10]: As @Hoikas commented, upstream is responsible for their target names -- vcpkg should not break the official usage guidance.
```suggestion
```

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
Thanks for the PR!

We would be happy to accept just the change to `vcpkg_from_github()`, but not modifying the targets.

File: ports/portaudio/portfile.cmake (hunk 1 above)
Comment (h3ndrk) [reply ports/portaudio/portfile.cmake]: Understood. Thanks for the feedback.

**Comment by h3ndrk** [comment]
Thanks for the feedback. I opened https://github.com/PortAudio/portaudio/pull/583 and reverted the namespace patch. The CI seems to hang...

**Review by PhoebeHui — APPROVED** [review]

File: ports/portaudio/portfile.cmake [hunk 2]
```diff
@@ -1,44 +1,45 @@
...
-endif()
-
-# Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+vcpkg_fail_port_install(ON_TARGET "uwp")
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO PortAudio/portaudio
+    REF 147dd722548358763a8b649b3e4b41dfffbcfbb6
+    TAG v19.7.0
```
Comment (strega-nil-ms) [inline ports/portaudio/portfile.cmake:6]:
```suggestion
```

**Review by strega-nil-ms — COMMENTED** [review]
Only minor changes required, thanks @h3ndrk :)

File: ports/portaudio/portfile.cmake (hunk 2 above)
Comment (h3ndrk) [reply ports/portaudio/portfile.cmake]: Oops, done.

**Review by PhoebeHui — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks!

**Pull request merged**